lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -223,15 +224,25 @@ def start(self):
                 self.processor.calculate_calibration_coefficients(event)
 
                 # write calib and pixel status
-                self.log.debug("Write pixel_status data")
+                self.log.info("Write pixel_status data")
                 self.writer.write('pixel_status', status_data)
 
-                self.log.debug("Write calibration data")
+                self.log.info("Write calibration data")
                 self.writer.write('calibration', calib_data)
+                self.n_calib += 1
+
                 if self.one_event:
                     break
 
     def finish(self):
+
+        self.log.info(f"Written {self.n_calib} calibration events")
+        if self.n_calib == 0:
+             self.log.critical(f"!!! No calibration events in the output file !!! : ")
```

Comment:
remove f

Reply:
Also: CRITICAL is enough, don't need all those exclamation marks